### DISCUSSION TITLE
Changing the dependencyDashboardTitle configuration causes renovate to create a new dashboard issue and not closing the old one

### DISCUSSION THREAD

**Discussion by so1ve** [renovatebot/renovate#21069] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

This is a repo which extends [my base renovate config](https://github.com/so1ve/renovate-config): https://github.com/so1ve/dlvr. There are two dependency dashboard issues.

I changed the dependencyDashboardTitle configuration several days ago, and a new dashboard is created and the old still exists.

I'm not sure whether this issue is caused by extending the base configuration.

### What you tried so far.

N/A

### Relevant debug logs

<details><summary>Logs</summary>

```
DEBUG: Ensuring Dependency Dashboard
DEBUG: ensureIssue(🍳 Renovate Dependency Dashboard)
DEBUG: Issue is open and up to date - nothing to do
```

The log does not make sense. I think renovate is just forgot to close the old issue.

</details>

**Comment by rarkins** [comment, marked as answer]
At the moment this is expected behavior. I have created an issue for the feature: #21072

**Reply by so1ve** [reply]
Thanks!
